lib/screens/playlist_gallery.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import '../widgets/gallery_view.dart';
+import '../widgets/sidebar.dart';
+
+class PlaylistGallery extends StatelessWidget {
+  static const List<String> _exerciseList = [
+    'Exercise 1',
```

Comment:
I think Lily gave names to the dummy data we're using. Could you change this to match those names?